api/v1alpha1/juicefsruntime_types.go

```diff
@@ -0,0 +1,236 @@
+/*
+Copyright 2021 The Fluid Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/fluid-cloudnative/fluid/pkg/common"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// JuiceFSRuntimeSpec defines the desired state of JuiceFSRuntime
+type JuiceFSRuntimeSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// The version information that instructs fluid to orchestrate a particular version of JuiceFS.
+	JuiceFSVersion VersionSpec `json:"juicefsVersion,omitempty"`
+
+	// The spec of init users
+	InitUsers InitUsersSpec `json:"initUsers,omitempty"`
+
+	// The component spec of JuiceFS worker
+	Worker JuiceFSCompTemplateSpec `json:"worker,omitempty"`
+
+	// The component spec of JuiceFS job Worker
+	JobWorker JuiceFSCompTemplateSpec `json:"jobWorker,omitempty"`
```

Comment:
Does Juice also have jobWorker?

Reply:
DataLoad need job to accomplish data warm-up. DataLoad is what we plan to do next.